packages/sdk/tests/key-management.unit.test.ts

```
@@ -0,0 +1,276 @@
+        });
+
+        test('should reject wrong length private key', () => {
+            const wrongLengthKey = new Uint8Array(16);
+            expect(keyManager.validatePrivateKey(wrongLengthKey)).toBe(false);
+        });
+    });
+
+    describe('Transaction Signing', () => {
+        test('should sign transaction correctly', () => {
+            const account = keyManager.createAccount(0);
+            
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)